src/components/FeaturedSection/FeaturedSection.tsx

```diff
@@ -0,0 +1,24 @@
+import Grid from "@mui/material/Grid";
+import styles from "./FeaturedSection.module.css";
+
+type Props = {
+  title: string;
+  callToAction: string;
+  children?: JSX.Element | JSX.Element[];
```

Comment:
Btw, you can just declare the children type as `ReactNode`. it's the same as this but shorter